Library/Formula/gsettings-desktop-schemas.rb

```diff
@@ -26,6 +26,7 @@ def install
                           "--disable-schemas-compile",
                           "--enable-introspection=yes"
     system "make install"
+    rm_f share/"glib-2.0/schemas/gschemas.compiled"
   end
 
   def post_install
```

Comment:
Looks like this is failing, not sure if it's related: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/24793/version=yosemite/testReport/junit/brew-test-bot/yosemite/install___gsettings_desktop_schemas_3_14_0_yosemite_bottle_2_tar_gz/